src/backend/renderer/multigpu/mod.rs

```diff
@@ -1224,6 +1240,10 @@ where
                         )
                         .map_err(Error::Target)?;
                     let sync = frame.finish().map_err(Error::Target)?;
+                    render
```

Comment:
Just to clarify, why does this call `cleanup_texture_cache()` explicitly, even though the new docs say that `Frame::finish()` does it automatically?

Reply:
The `frame` here is from the `target`, so the call to `cleanup_texture_cache` should clean-up the other renderer in this case.

Reply:
I see, thanks